cmd/virsnap/root.go

```diff
@@ -2,44 +2,61 @@
 // Licensed under the MIT License. You have obtained a copy of the License at
 // the "LICENSE" file in this repository.
 
-// Package cmd implements the handlers for the different command line arguments.
-package cmd
+// Package main implements the handlers for the different command line arguments.
+package main
 
 import (
-	"github.com/joroec/virsnap/pkg/virt"
+	"fmt"
+	"os"
+
+	"github.com/joroec/virsnap/pkg/instrument/log"
 	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var (
+	// RootCmd is a global variable defining the corresponding cobra command.
+	RootCmd = &cobra.Command{
+		Use: "virsnap",
+		Short: "virsnap is a small tool that eases the automated creation and " +
+			"deletion of VM snapshots.",
+		Long: "virsnap is a small tool that eases the automated creation and " +
+			"deletion of VM snapshots.",
+		PersistentPreRun: initLogger,
```

Comment:
Could you please add a comment on why we call initLogger via PersistentPreRun and not via default init (Command line parsing needs to be finished before logger gets initialized properly)?

Reply:
Done